dmci/distributors/distributor.py

```diff
@@ -117,6 +117,7 @@ def run(self):
     def is_valid(self):
         return self._valid
 
+    # Should be possible to remove this is we consistently use uri:UUID
```

Comment:
Should this comment be removed? I think my idea was to use the combination of namespace and uuid as identification, but it is maybe also ok like it is done now. In that case the comment can be deleted.
```suggestion
```